clients/test/lightrider-entry-test.js

```diff
@@ -119,17 +119,21 @@ describe('lightrider-entry', () => {
       runStub.mockRestore();
     });
 
+    let originalRun;
```

Comment:
#9828 disallowed these functions from being nested in a test case.

Reply:
oh dang yeah, I did not expect that to work haha